hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/grpc/metrics/GrpcMetrics.java

```diff
@@ -100,6 +101,8 @@ public static synchronized GrpcMetrics create(Configuration conf) {
    */
   public void unRegister() {
     DefaultMetricsSystem.instance().unregisterSource(SOURCE_NAME);
+    MetricUtil.stop(grpcProcessingTimeMillisQuantiles);
+    MetricUtil.stop(grpcQueueTimeMillisQuantiles);
```

Comment:
```suggestion
    MetricUtil.stop(grpcProcessingTimeMillisQuantiles, grpcQueueTimeMillisQuantiles);
```

Reply:
Thanks @jojochuang for the review.

Both `grpcProcessingTimeMillisQuantiles` and `grpcQueueTimeMillisQuantiles` are arrays, so this does not work.